src/test/java/org/folio/innreach/controller/InnReachRecallUserControllerTest.java

```diff
@@ -0,0 +1,118 @@
+package org.folio.innreach.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlMergeMode;
+
+import org.folio.innreach.controller.base.BaseControllerTest;
+import org.folio.innreach.dto.InnReachRecallUserDTO;
+import org.folio.innreach.repository.CentralServerRepository;
+
+@Sql(scripts = {
+  "classpath:db/central-server/clear-central-server-tables.sql",
+  "classpath:db/inn-reach-recall-user/clear-inn-reach-recall-user.sql"
+}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+@SqlMergeMode(SqlMergeMode.MergeMode.MERGE)
+class InnReachRecallUserControllerTest extends BaseControllerTest {
+
+  @Autowired
+  private CentralServerRepository centralServerRepository;
+
+  @Autowired
+  private TestRestTemplate testRestTemplate;
+
+  @Test
+  @Sql(scripts = {
+    "classpath:db/inn-reach-recall-user/pre-populate-inn-reach-recall-user.sql",
+    "classpath:db/central-server/pre-populate-central-server-with-recall-user.sql"
+  })
+  void returnRecallUser_when_centralUserRecallUserExists() {
+    var responseEntity = testRestTemplate.getForEntity("/inn-reach/central-servers/{id}/inn-reach-recall-user",
+        InnReachRecallUserDTO.class, UUID.fromString("edab6baf-c696-42b1-89bb-1bbb8759b0d2"));
+
+    assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+
+    var body = responseEntity.getBody();
+
+    assertNotNull(body);
+  }
+
+  @Test
+  @Sql(scripts = {
+    "classpath:db/inn-reach-recall-user/pre-populate-inn-reach-recall-user.sql",
+    "classpath:db/central-server/pre-populate-central-server-with-recall-user.sql"
+  })
+  void returnErrorMessage_when_centralServerRecallUserDoesNotExist() {
+    var responseEntity = testRestTemplate.getForEntity("/inn-reach/central-servers/{id}/inn-reach-recall-user",
+        InnReachRecallUserDTO.class, UUID.fromString("edab6baf-c697-42b2-89bb-1bbb8759b0d3"));
```

Comment:
the same request path and uuid are used among multiple tests, it's better to set them as constants